lib/serve/rest-api/src/requirements.txt

```diff
@@ -1,10 +1,13 @@
 aiobotocore
```

Comment:
Not directly related to this PR but we need to pin these.

Reply:
Not a problem, I'll generate those